app/decorators/user_decorator.rb

```diff
@@ -6,14 +6,17 @@ def twitter_url
   end
 
   def roles
-    roles = []
-
-    roles << :retired if retired_on?
-    roles << :admin if admin?
-    roles << :mentor if mentor?
-    roles << :adviser if adviser?
-    roles << :graduate if graduated_on?
-    roles << :trainee if trainee?
+    role_list = [
+      { role: 'retired', value: retired? },
+      { role: 'hibernationed', value: hibernated? },
+      { role: 'admin', value: admin },
```

Comment:
booleanを返すところが重要なので元のコードにあるように`?`をつけた書き方がいいかもです〜

```suggestion
      { role: 'admin', value: admin? },
```

Reply:
確かにこちらの方がbooleanを返すことが想像しやすいですね！🤲